Pipeline/CreateUCDComponentVersion/dbb-ucd-packaging.groovy

```diff
@@ -124,7 +142,14 @@ def xml = new MarkupBuilder(writer)
 xml.manifest(type:"MANIFEST_SHIPLIST"){
 	//top level property will be added as version properties
 	//requires UCD v6.2.6 and above
+	// Url to DBB Build result
 	property(name : buildResult.getGroup() + "-buildResultUrl", value : buildResult.getUrl())
+	// Url to CI pipeline
+	if (properties.pipelineURL)property(name : "ci-pipeline-url", value : properties.pipelineURL )
+	// Populate build result properties
+	if (buildResultProperties != null) buildResultProperties.each{
+		property(name:it.key, value:it.value)
+	}
```

Comment:
@dennis-behm 
Just something that come in my mind. Senthil pinged us in the past saying that when properties in this area have specific names they are duplicated from version to version. Maybe you can handle this in your PR?
![image](https://user-images.githubusercontent.com/60784175/121870058-f3de6880-cd02-11eb-88a0-e6ffc2ba7a44.png)

Reply:
@jbyibm yes, this is something, I can include in this as well. Capturing the gitBranch as an optional property might also be interesting.

The buildResultProperties keys should not vary, so we should be ok with publishing them in there.

Can you have another look?